src/components/Footer.test.js

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from "@testing-library/react"
+import { BrowserRouter } from "react-router-dom"
+import userEvent from "@testing-library/user-event"
+import Footer from "./Footer"
+
+describe("<Footer/>", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div")
+    render(
+      <BrowserRouter>
+        <Footer />
+      </BrowserRouter>,
+      div
+    )
+  })
+  it("renders a logo with a src and alt", () => {
+    const div = document.createElement("div")
+    render(
+      <BrowserRouter>
+        <Footer />
+      </BrowserRouter>,
+      div
+    )
+  })
+  it("has clickable links", () => {
+    render(
+      <BrowserRouter>
+        <Footer/>
+      </BrowserRouter>
+    )
+  })
```

Comment:
There are no tests here just renders. 

Please come up wit an action or an assertion for each of your statements.